conans/model/ref.py

```diff
@@ -5,6 +5,16 @@
 from conans.model.version import Version
 
 
+def check_valid_ref(ref, allow_pattern=False):
```

Comment:
allow_pattern is always False, then so far it is a bit speculative, isn't it?

Reply:
Maybe it is better not to provide a default, I'll change it.